test/plugin/scenarios/rocketmq-scenario/config/expectedData.yaml

```diff
@@ -62,6 +62,8 @@ segmentItems:
             peer: ''
             tags:
               - {key: transmission.latency, value: not null}
+              - { key: mq.topic, value: TopicTest }
+              - { key: mq.broker, value: not null }
```

Comment:
It's  recommended to use `not  blank` rather than `not null ` when validating tags .
Refer to  https://github.com/apache/skywalking-java/pull/362

Reply:
It seems contains a bug when using `not blank `. The validator.out log like below
```
  Segment[a7cbd27526b941c1b22a22a6b6f6b85e.38.16670159779570000] e
  expected:	Span[0, 1] RocketMQ/TopicTest/Producer
  actual:	span[0, 1] RocketMQ/TopicTest/Producer
  reason:	[tag(mq.broker) value]: expected=>{not blank}, actual=>{172.23.0.2:10911}
```

Reply:
I found out why, you support not blank validator on 2022/10/21 commit on skywalking-agent-test-tool. I will change the `agent-test-tools.version` and try again.

Reply:
Yes ,  merging master into your repo could resolve this

Reply:
> Yes , merging master/main branch into your repo could resolve this

Resolved, and `not blank` commit pushed.